tests/datatypes/test_dict.py

```diff
@@ -20,6 +20,12 @@ def test_getattr(self):
                 print(err)
             """)
 
+    def test_copy(self):
+        self.assertCodeExecution("""
+            x = {'a': 1, 'b': 2, 'c': 3}
+            print(x.copy())
```

Comment:
This is not a complete test for `dict.copy()`, nor the implementation does the right thing (it's returning the same dict, instead of creating a new one).

You should test it with:

```
x = {'a': 1}
y = x.copy()
y['b'] = 2
print(x)
print(y)
```